test/fork.js

```diff
@@ -38,11 +38,9 @@ test('rejects on error and streams output', function (t) {
 
 	fork(fixture('broken.js'))
 		.run()
-		.on('uncaughtException', function (data) {
-			t.true(/no such file or directory/.test(data.exception.message));
-		})
-		.catch(function () {
-			t.pass();
+		.catch(function (err) {
+			t.true(/exited with a non-zero exit code: 1/.test(err.message));
+			t.ok(err);
```

Comment:
Are you talking about this test here? If this test passes, that seems fine to me. We are getting a failure. Users of the CLI or API will get "can't find any files to test", like they have. Anybody using `lib/fork` from outside AVA is already asking for trouble.

We need to investigate why it fails on Node 0.10.